common_design_subtheme/sass/base/_typography.scss

```diff
@@ -3,10 +3,10 @@
 //——————————————————————————————————————————————————————————————————————————————
 
 a {
-  color: $cd-blue--dark;
+  color: var(--cd-blue--dark);
 
   &:hover,
   &:focus {
-    color: $cd-ocha-blue;
+    color: var(--cd-ocha-blue);
   }
 }
```

Comment:
Up to this point, I had been trying to avoid any changes to the styles themselves, instead focusing on swapping the Sass variables for CSS Vars. But I agree there are other places where we can remove duplicates and otherwise streamline.